source/developer_manual/design/doveadm_http_protocol.rst

```diff
@@ -0,0 +1,113 @@
+.. _dovecot_doveadm_http_protocol:
```

Comment:
isn't this already documented elsewhere?

Reply:
yeah, i removed it and linked to the existing pages.